content/en/docs/concepts/workloads/controllers/replicaset.md

```diff
@@ -310,6 +310,18 @@ assuming that the number of replicas is not also changed).
 A ReplicaSet can be easily scaled up or down by simply updating the `.spec.replicas` field. The ReplicaSet controller
 ensures that a desired number of Pods with a matching label selector are available and operational.
 
+When scaling down, the ReplicaSet controller chooses which pods to delete by sorting the available pods to
+prioritize scaling down pods based on the following general algorithm:
+ 1. Pending (and unschedulable) pods are scaled down first
+ 2. If controller.kubernetes.io/pod-deletion-cost annotation is set, then
+    the pod with the lower value will come first.
+ 3. If the pods' ranks differ, the pod with greater rank comes before the pod
```

Comment:
rank is an internal detail to the controller utils. The rank we use in replicaset controller is the number of pods in a node.

Reply:
Would this be better as just something like `3. Pods on nodes with greater replicas come before pods on nodes with fewer replicas`?

Reply:
Sounds good. But should it be `greater` or `more`? Or perhaps `greater number`?

Reply:
Updated with `more` to keep it clear :)